project/ConsoleTracker/ConsoleTrackerApp/tasks.py

```diff
@@ -82,3 +130,7 @@ def start_query_daemon():
     t = Thread(target=query_time)
     t.daemon = True
     t.start()
+
+    t2 = Thread(target=update_time_thread)
+    t2.daemon = True
+    t2.start()
```

Comment:
is it better to create another daemon for this? 
someone added another periodic function within the existing background thread, so shouldn't we do the same here

Reply:
I didn't want to add to the complexity of the existing function (also didn't want to touch your code, lol). I didn't want to make the code too difficult to follow, but there is reason to keep it in the same thread. With how I currently have it set up I'm worried about race conditions on accessing the expired data on the User_uses_machine model.

I can go ahead and refactor the code to separate the different tasks in the thread so it looks cleaner. Let me know what you think.

Reply:
Yea that makes sense. Refactoring the code that looks for expired data into a different function, then adding your update time function before it sounds good here.